docs-2.0/reuse/source_manage-service.md

```diff
@@ -1,14 +1,28 @@
-You can use the `nebula.service` script to start, stop, restart, terminate, and check the Nebula Graph services. This topic takes starting, stopping and checking the Nebula Graph services for examples.
+Nebula Graph supports managing services with script or systemd. This topic will describe the two methods in detail.
 
-`nebula.service` is stored in the `/usr/local/nebula/scripts` directory by default, which is also the default installation path of Nebula Graph. If you have customized the path, use the actual path in your environment.
+!!! enterpriseonly
 
-## Syntax
+    Systemd only available for the Nebula Graph Enterprise Edition.
+
+!!! note
+
+    The two methods are incompatible. It is recommended to use only one method in a cluster.
+
+## Manage services with script
+
+You can use the `nebula.service` script to start, stop, restart, terminate, and check the Nebula Graph services.
+
+!!! note
+
+    `nebula.service` is stored in the `/usr/local/nebula/scripts` directory by default, which is also the default installation path of Nebula Graph. If you have customized the path, use the actual path in your environment.
```

Comment:
```suggestion
    `nebula.service` is stored in the `/usr/local/nebula/scripts` directory by default. If you have customized the path, use the actual path in your environment.
```

`/usr/local/nebula/` is the default installation path, but `/usr/local/nebula/scripts` is not.